src/notation/view/widgets/editstyle.ui

```diff
@@ -11867,7 +11873,8 @@ text x-height):</string>
                       <string>Reset to default</string>
                      </property>
                      <property name="accessibleName">
-                      <string>Reset 'Dash Y position ratio' value</string>
+                      <string>
+                        Reset 'Vertical position (% of text x-height)' value</string>
```

Comment:
A newline has appeared here; I'm not sure whether that might have adverse effects

Reply:
OOps. Shouldn#t matter much, as this is for screen readers, but let's fix it anyhow